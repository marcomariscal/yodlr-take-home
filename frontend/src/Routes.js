import React from "react";
import { Redirect, Route, Switch } from "react-router-dom";
import Home from "./Home";
import SignUp from "./SignUp";
import Admin from "./Admin";

const Routes = () => {
  return (
    <div>
      <Switch>
        <Route exact path="/">
          <Home />
        </Route>
        <Route exact path="/signup">
          <SignUp />
        </Route>
        <Route exact path="/admin">
          <Admin />
        </Route>
        <Redirect to="/" />
      </Switch>
    </div>
  );
};

export default Routes;
