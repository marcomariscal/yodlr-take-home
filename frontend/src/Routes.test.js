import React from "react";
import { render } from "@testing-library/react";
import Routes from "./Routes";
import { MemoryRouter } from "react-router";

it("renders without crashing", function () {
  render(
    <MemoryRouter>
      <Routes />
    </MemoryRouter>
  );
});

it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <Routes />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
