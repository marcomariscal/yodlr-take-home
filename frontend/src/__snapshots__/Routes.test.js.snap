// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="Home"
    >
      <h1>
        Yodlr Design Challenge!
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
